src/cmd-cloud-prune

```diff
@@ -162,6 +169,19 @@ def main():
                         case "build":
                             prune_build(s3_client, bucket, prefix, build_id, args.dry_run)
                             pruned_build_ids.append(build_id)
+                        case "containers":
```

Comment:
since containers are manifest listed let's add a conditional here and only act when arch == x86_64

Reply:
resolved in new upload